source/easeOfAccess.py

```diff
@@ -16,8 +16,6 @@
 import winVersion
 
 
-# Windows >= 8
-canConfigTerminateOnDesktopSwitch: bool = winVersion.getWinVer() >= winVersion.WIN8

```

Comment:
I think we should just set this to `True` and deprecate. It seems easy to maintain this as an alias for add-ons to give authors time to remove win7/8 code in their own time.

Reply:
Honestly, I think it is very unlikely that this is used in add-on. If an add-on author would like to check for specific versions of Windows, he's supposed to use the winVersion module anyway.

I'm happy to restore it though.

Reply:
Our current policy is to only break the API when necessary, and maintain aliases where possible and maintenance is minimal